app/controllers/idv/personal_key_controller.rb

```diff
@@ -38,6 +39,14 @@ def update
 
     private
 
+    def opt_in_analytics_properties
+      extra = {}
+      if IdentityConfig.store.in_person_proofing_opt_in_enabled
+        extra = { opted_in_to_in_person_proofing: idv_session.opted_in_to_in_person_proofing }
+      end
+      extra
```

Comment:
same thing about removing this assignment.
Since this is the 3rd instance, maybe woudl this make for a good Mixin module?
```suggestion
      if IdentityConfig.store.in_person_proofing_opt_in_enabled
        { opted_in_to_in_person_proofing: idv_session.opted_in_to_in_person_proofing }
      else
        {}
      end
```